proto/parca/metastore/v1alpha1/metastore.proto

```diff
@@ -298,4 +313,10 @@ message Mapping {
 
   // has_inline_frames indicates whether the mapping has associated inline frames.
   bool has_inline_frames = 10;
+
+  // fileStringIndex is the index in the string table to the file name associated with the mapping.
+  uint32 file_string_index = 11;
```

Comment:
curious to what the difference is between `filename_string` and `file_string`

Reply:
This is just following whatever the meta structure already had, adding a `_string_index` suffix.